scripts/Muon/GUI/Common/contexts/fitting_context.py

```diff
@@ -385,6 +385,17 @@ def find_output_workspaces_for_input_workspace_name(
 
         return workspace_list
 
+    def find_fit_for_input_workspace_list(
+            self, input_workspace_list):
+        """
+        Find the fits in the list whose input workspace matches
+        :param input_workspace_name: The name of the input_workspace
+        :return: A list of matching fits
+        """
+        for fit in self.fit_list:
+            if fit.input_workspaces == input_workspace_list:
+                return fit
```

Comment:
this will only return a single fit. Is that correct?

Reply:
Yeah thats correct

Reply:
the description seems to suggest otherwise. Can we update it?